### PR TITLE
Move tide whisperer

### PR DESCRIPTION
**Pull request by derrickburns**
Fix broken helm chart.

### CODE REVIEW

File: charts/tidepool/0.1.7/charts/messageapi/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Kubernetes
-name: message-api
+name: messageapi
```
Comment (cbwebdevelopment) [inline charts/tidepool/0.1.7/charts/messageapi/Chart.yaml]: I noticed that to correspond with this change, you've renamed some files to use the hyphen-free counterpart, while others were not updated.  (same with the `tide-whisperer -> tidewhisperer` change)

Specifically, it seems that files in the `templates` directory were unchanged, and those in the `charts` directory were.  Would it be better to have this consistent everywhere?

File: cmd/tpctl.sh [hunk 2]
```diff
@@ -628,14 +628,15 @@ EOF
 # update flux and helm operator manifests
 function update_flux {
         start "updating flux and flux-helm-operator manifests"
+	local config=$(get_config)
```
Comment (cbwebdevelopment) [inline cmd/tpctl.sh]: Small nitpick on whitespace here -- should align with the rest of the block

File: charts/tidepool/0.1.7/charts/messageapi/Chart.yaml (hunk 1 above)
Comment (derrickburns) [reply charts/tidepool/0.1.7/charts/messageapi/Chart.yaml]: Yes. I will make that change in an upcoming PR to get this through.

File: cmd/tpctl.sh (hunk 2 above)
Comment (derrickburns) [reply cmd/tpctl.sh]: Thx

**Pull request merged**